src/methods/alphabetical-sort.js

```diff
@@ -34,8 +34,11 @@ const alphabeticalSort = list => {
  * @returns {array}
  *
  * @example
```

Comment:
for this method I think would be good to have another example block for list of objects, the `list` param (the only function param) may also be a list of nested objects (an array of products, brands or any...), then the objects will be sorted inside array using the name/title prop as reference.
imo it's important to have an example with that, wdyt?